pydis_site/apps/content/resources/guides/pydis-guides/contributing/docker.md

```diff
@@ -0,0 +1,119 @@
+
+Consider reading some of the following topics if you're interested in learning more about Docker itself:
+
+ * [**What is Docker?**](https://docs.docker.com/engine/docker-overview/)
+ * [**How can I learn to use it for my own stuff?**](https://docs.docker.com/get-started/)
+ * [**What about Docker Compose, what's it for?**](https://docs.docker.com/compose/)
+
+# Docker Installation
+You can find installation guides available for your respective OS from the official Docker documentation:
+[https://docs.docker.com/install/](https://docs.docker.com/install/)
+
+## After Installing on Linux
+If you're on Linux, there's a few extra things you should do:
+
+1. [**Add your user to the `docker` user group so you don't have to use `sudo` when running docker or docker-compose.**](#add-user-group)
+2. [**Start up the Docker service.**](#run-the-service)
+3. [**Set the Docker service to start on boot.**](#start-on-boot) **(optional)**
+
+### Run the Service
+Most linux distributions **systemd**, you can start the service with:
+```shell
+$ sudo systemctl start docker
+```
+
+### Add User Group
+```shell
+$ sudo groupadd docker
+$ sudo usermod -aG docker $USER
+```
+Log out and log back in to ensure your group changes work.
+
+### Start on Boot
+```shell
+$ sudo systemctl enable docker
+```
+
+# Possible Issues
+### Couldn't connect to Docker daemon
+```shell
+ERROR: Couldn't connect to Docker daemon at http+docker://localhost - is it running?
+```
+**Problem**  
+Your Docker service is either not started, or you haven't yet installed Docker.
+
+**Solution**  
+[Start the service](#run-the-service) or ensure it's installed.
+If it's not, [install it](#docker-installation).
+
+### Error loading config file
+```plaintext
+WARNING: Error loading config file: /home/user/.docker/config.json -
+stat /home/user/.docker/config.json: permission denied
+```
+**Problem**  
+You initially ran Docker using `sudo` before adding your user to the `docker` group, resulting in your `~/.docker/` directory being created with incorrect permissions.
+
+**Solution**  
+Remove the existing `~/.docker/` directory. It will be automatically re-created with the correct permissions.
+
+### Drive has not been shared (Windows users)
+
+When attempting to run the `docker-compose up` command on a Windows machine, you receive the following or similar error message:
+```text
+ERROR: for bot_bot_1 Cannot create container for service bot: b'Drive has not been shared'
+```
+**Problem**  
+Windows has not been configured to share drives with Docker.
+
+**Solution**  
+> NOTE: Solution requires Windows user credentials for an account that has administrative privileges.
+
+1. Right-click the Docker icon in the Windows system tray, and choose "Settings" from the context menu.
+![Docker Settings](/static/images/content/contributing/docker_settings.webp)
```

Comment:
This image has alignment issues like the other mentioned cases.

Reply:
Addressed in a782abd.